arch/lkl/kernel/time.c

```diff
@@ -29,6 +29,11 @@ void calibrate_delay(void)
 {
 }
 
+void read_persistent_clock(struct timespec *ts)
```

Comment:
It might make sense to make this public. One possible use case is syncing the clock with the host in case of drift in long-running application or a simple change of the host time.